docs/iot/TOC.md

```diff
@@ -38,7 +38,7 @@
 ### [Tizen Common Headed API Reference](https://developer.tizen.org/dev-guide/tizen-iot-headed/latest)
 ### [Tizen Common Headless API Reference](https://developer.tizen.org/dev-guide/tizen-iot-headless/latest)
 
-# Preview 2
+# Tizen IoT Preview 2(Tizen 4.0)
```

Comment:
Preview 2 is located under "IoT and Tizen IoT".
I think we don't need to make a title with repeated words. 
Moreover, "(Tizen 4.0)" is not needed into the menu title in my opinion.

If you want to change it, "Tizen IoT Preview 2" is acceptable.

Reply:
'Tizen IoT Preview 2" looks like a newer version for pre-release, so, I want to make it clear that Tizen IoT Preview 2 is not new and old version.

Reply:
If developers click the menu, they can see the overview page of Preview 2.
As soon as they see the page, they can also confirm the contents are based on Tizen 4.0.

I think developers won't be confused that Preview 2 is new version.
Didn't you insert the following note to overview.md?
```
       > **Note**
        >
        > The API set of Tizen IoT Extension SDK version 1 based on Tizen 5.0 is not compatible with that of Tizen IoT Preview 2 based on Tizen 4.0.
```

Reply:
"Preview 2" is not clear to developers. They may not understand that "Preview 2" is the same as "Tizen IoT Preview 2". And for the first visitor, it may be confused. I think it is better to clarify the title, even if there is a note or comment in the page they enter.

Reply:
I told you "Tizen IoT Preview 2" was acceptable.

Reply:
I add a note at the top of preview2/getting-started/overview.md.
> ** Note
> Tizen IoT Preview 2 is working only on Tizen 4.0. 
>

For the first developer, "Tizen IoT Preview 2" looks the next new version for experiment and test. I want to add (Tizen 4.0) in order to make it clear. What do you think?

Reply:
After brief discussion, we decided to write "Tizen Preview 2". I upload the patch.